### PR TITLE
first commit of alex

### PR DESCRIPTION
**Pull request by aasanchez**
This is what alex, made from portutal, dammm portugeuse

### CODE REVIEW

File: AlexandreG/AlexMathLib.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace alexandreg
+{
+    public static class AlexMathLib
+    {
+        public static long Sum(long num1, long num2)
+        {
+            return num1 + num2;
```
Comment (aasanchez) [inline AlexandreG/AlexMathLib.cs]: damm... never return a operation.....

**Comment by AngeloBadellino** [comment]
I see something

File: AlexandreG/AlexMathLib.cs (hunk 1 above)
Comment (aasanchez) [inline AlexandreG/AlexMathLib.cs]: do you check if the number were numbers? where is the validation

File: GitTraining/GitTraining.csproj [hunk 2]
```diff
@@ -5,4 +5,8 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\AlexandreG\AlexandreG.csproj" />
```
Comment (countDuca) [inline GitTraining/GitTraining.csproj]: What is this?

**Pull request closed without merging**